Hunted Mobile/Hunted Mobile/Hunted Mobile/Model/GameModels/Loot.cs

```diff
@@ -1,15 +1,42 @@
-﻿using System;
+﻿using Hunted_Mobile.ViewModel;
+
+using System;
 using System.Collections.Generic;
 using System.Text;
 
 namespace Hunted_Mobile.Model.GameModels {
-    public class Loot {
+    public class Loot : BaseViewModel {
         private readonly int _id;
+        private bool _isHandlingLoot = false;
+        private bool _hasFinishedHandlingLoot = false;
+
         public Location Location { get; set; }
         public string Name { get; set; }
 
         public Loot(int id) {
             _id = id;
         }
+
+        public bool IsHandlingLoot {
```

Comment:
Ik heb deze public boolean in het `loot` model geplaatst, want anders werd de MapViewModel enorm vol. Als dit niet de bedoeling is, dan plaats ik het terug.